python/test/CMakeLists.txt

```diff
@@ -342,6 +342,7 @@ ot_pyinstallcheck_test (KissFFT_std)
 ot_pyinstallcheck_test (FilonQuadrature_std)
 ot_pyinstallcheck_test (GaussKronrod_std)
 ot_pyinstallcheck_test (GaussLegendre_std)
+ot_pyinstallcheck_test (FejerAlgorithm_std)
```

Comment:
you can use IGNOREOUT and remove the expout as its empty

Reply:
Indeed